src/main/kotlin/org/rust/ide/annotator/fixes/AddMutableFix.kt

```diff
@@ -32,10 +32,18 @@ class AddMutableFix(binding: RsNamedElement) : LocalQuickFixAndIntentionActionOn
     companion object {
         fun createIfCompatible(expr: RsExpr): AddMutableFix? {
             val declaration = expr.declaration
```

Comment:
Maybe perform cast to `RsNamedElement` here? Like `val declaration = expr.declaration as? RsNamedElement ?: return null`